thirdeye/thirdeye-frontend/app/utils/manage-alert-utils.js

```diff
@@ -306,6 +306,28 @@ export function getDuration() {
   return JSON.parse(localStorage.getItem('duration'));
 }
 
+/**
+ * Caches metric data to local storage in order to persist it across pages
+ * NOTE: not in use yet
+ * @method setMetricData
+ * @param {Number} metricId - id of selected metric
+ * @param {Object} metricData - payload for graphable metric data
+ * @return {undefined}
+ */
+export function setMetricData(metricId, metricData) {
```

Comment:
This is an experiment with metric data caching - not in use yet, as there are better ways of doing this.